general-superstaq/general_superstaq/service.py

```diff
@@ -1,14 +1,15 @@
 import os
-from typing import Any, Dict, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Union
 
-import general_superstaq as gss
+import qubovert as qv
 
 
-class UserConfig:
+
+class Service:
     """This class contains all the user configurations that are used to operate Superstaq."""
```

Comment:
This probably needs an updated docstring. I'm assuming `solve_qubo` falls outside the umbrella of simply setting using configurations.

Reply:
A lot of stuff broke, but we were able to fix the issues. Docstrings are updated (minor tweak). Done.